.github/workflows/update-test-images.yml

```diff
@@ -0,0 +1,31 @@
+name: Update Test Images
+on:
+  push:
+    branches: ['terraform']
+
+    # List of paths that should be considered when triggering the workflow
+    paths: ['mocked_servers/**', 'sample-apps/**']
+jobs:
+  run-image-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          role-to-assume: ${{ secrets.TEST_FW_ASSUMABLE_ROLE_ARN }}
```

Comment:
When using an OIDC provider the default session length is 1 hour. Are we going to need to set it to be longer for this? 

[ref](https://github.com/aws-actions/configure-aws-credentials#assuming-a-role)